src/main/java/net/minecraftforge/event/entity/LightningStrikeEvent.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Minecraft Forge
+ * Copyright (c) 2016-2018.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.event.entity;
+
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.ForgeEventFactory;
+import net.minecraftforge.fml.common.eventhandler.Cancelable;
+import net.minecraft.entity.effect.EntityLightningBolt;
+
+/**
+ * LightningStrikeEvent is fired when a lightning entity has struck a block.<br>
```

Comment:
Why do you have two different explanations as to when this event fires? Is it when it hits a block or an entity? Is it fired once per entity, or once per strike? Rephrase this whole thing to be clearer.

Reply:
```suggestion
  * LightningStrikeEvent is fired once when a lightning entity strikes.<br>
  * This event is fired whenever an EntityLightningBolt strikes
```
What about this?

Reply:
@tterrag1098 I've updated the description to be a lot clearer.